mozilla/fx-private-relay
# MPP-2359 - Toast notification after resending welcome text

# New feature description

This adds a toast notification and dismissal to the resend welcome text banner after the CTA is clicked.

# Screenshot (if applicable)

https://user-images.githubusercontent.com/13066134/192007943-7a985ba4-b877-4c3e-a8e4-d51462229c46.mov


# How to test
On the phones dashboard page, check that the clicking the "Resend welcome text" CTA closes the banner, and displays a toast notification thereafter. Make sure that the contact card is still relayed normally to your real phone number.

# Checklist
